### DISCUSSION TITLE
완료한 챌린지에 대해 체크 표시가 되어 있지 않습니다.

### DISCUSSION THREAD

**Discussion by minji1289** [devrel-kr/HackaLearn#258] [1 Q&A - HackaLearn 운영] [answered]
안녕하세요, 팀 토킹 포테이토입니다. 
<img width="390" alt="스크린샷 2021-08-15 오후 2 38 27" src="https://user-images.githubusercontent.com/79117648/129468381-5fceb1a4-2b23-4e01-aecf-2391faaefa6c.png">
위와 같이 완료한 챌린지(#1,#2,#3)임에도 불구하고 체크리스트에 체크 표시가 되어있지 않아 질문 드립니다. 감사합니다.

**Comment by justinyoo** [comment, marked as answer]
@minji1289 이 파일은 저희가 건드리면 안되구요, 팀 멤버분들께서 수정해 주셔야 하는 파일이예요. 따라서 새 PR을 생성하셔서 위의 체크박스를 수정해 주시면 됩니다. 체크 박스는 현재 `[ ]` 모양일텐데, 이를 `[x]`로 바꾸면 체크박스로 표시됩니다. PR 생성하실 때 코멘트로 이 부분은 체크박스 부분만 수정하는 거라고 해 주시면 저희가 곧바로 머지하도록 하겠습니다.

**Reply by minji1289** [reply]
답변 정말 감사합니다!!
